docs/integrations/flutter-specific-workflow-steps/flutter-test.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Flutter Test
+metaTitle: Flutter Test
+metaDescription: Flutter Test
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Flutter Test
+
+This component allows you to run [**Flutter Unit Tests**](https://docs.flutter.dev/cookbook/testing/unit/introduction#run-tests-in-a-terminal). Please note that it requires the preceding [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) and [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) steps to run.
```

Comment:
> Please note that it requires the preceding [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) and [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) steps to run

Is there a reason to repeat here exactly the same dependencies that we wrote at **Prerequisites**? If not, it might be better to delete it for simplicity.